### PR TITLE
Dp 1044

### PR DESCRIPTION
**Pull request by JordanBalesBAO**
### Related Issues
- DP-XXX
- etc.

## Reviewer:
- [ ] Tests added/updated & passing.
- [ ] Clean linting.
- [ ] Related issue ticket in Jira/GitHub.
- [ ] Documentation added/updated.
- [ ] Code conforms to style guidelines.
- [ ] Well commented.
- [ ] Updates reflected in NEWS.md.
- [ ] Build check passes.

### CODE REVIEW

File: R/unPackYear2Sheet.R [hunk 1]
```diff
@@ -610,9 +610,13 @@ unpackYear2Sheet <- function(d) {
       attributeOptionCombo,
       value = value
     ) %>%
-    tidyr::drop_na() %>%
+    # tidyr::drop_na() %>% # Tue Dec  5 14:19:52 2023 delete if 2 months pass
     dplyr::distinct()
 
+  if (any(is.na(d$datim$year2))) {
+    warning("NA's found in final Year2 export.")
+  }
```
Comment (jason-p-pickering) [inline R/unPackYear2Sheet.R:618]: Hmm...if there are any NAs do we not drop them? I think we should

Comment (JordanBalesBAO) [reply R/unPackYear2Sheet.R:618]: Added back the drop NA line. Working on adding to messages

**Comment by JordanBalesBAO** [comment]
Enterd https://jira.pepfar.net/browse/DP-1182 to handle the testing for na's piece of this puzzle.

**Comment by JordanBalesBAO** [comment]
line 156 is all that changed for the KP regex not to get dumped.

**Review by flopez-bao — APPROVED** [review]

**Pull request merged**